ads/fusion.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright 2016 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {writeScript, validateData} from '../3p/3p';
+
+/**
+ * @param {!Window} global
+ * @param {!Object} data
+ */
+export function fusion(global, data) {
+  validateData(data, [], ['mediaZone', 'layout', 'adServer', 'space']);
+
+  const container = global.document.getElementById('c');
+  const ad = global.document.createElement('div');
+  ad.setAttribute('data-fusion-space', data.space);
+  container.appendChild(ad);
+
+  writeScript(global, 'https://assets.adtomafusion.net/fusion/0.9/latest/fusion-default.js', () => {
+    global.Fusion.apply(container, global.Fusion.loadAds(data));
```

Comment:
please add the `global.Fusion`, `data.space`, `data.mediaZone` ... to the `ads.extern.js` file to pass the type check.